public/pages/Dashboard/utils/utils.tsx

```diff
@@ -0,0 +1,369 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from 'react';
+import { EuiLink, EuiText, EuiTextAlign } from '@elastic/eui';
+import {
+  AD_DOC_FIELDS,
+  SORT_DIRECTION,
+} from '../../../../server/utils/constants';
+import { Detector, FeatureAttributes } from '../../../models/interfaces';
+import { PLUGIN_NAME } from '../../../utils/constants';
+
+/**
+ * Get the recent anomaly result query for the last timeRange period(Date-Math)
+ * Given timeRange is 24h, it return a query which is used to get anomaly result
+ * within last 24 hours
+ * @param  {[string]} timeRange [last time period which query is for]
+ * @returns query which is used to get anomaly result for the last timeRange period
+ */
+export const buildGetRecentAnomalyResultQuery = (timeRange: string) => {
+  return {
+    range: {
+      [AD_DOC_FIELDS.DATA_START_TIME]: {
+        gte: 'now-' + timeRange,
+      },
+    },
+    size: 30,
+    sortField: AD_DOC_FIELDS.DATA_START_TIME,
+    from: 0,
+    sortDirection: SORT_DIRECTION.DESC,
+  };
+};
+
+export type RgbColor = [number, number, number, number?];
+export const rgbColors: RgbColor[] = [
```

Comment:
Looking at their utils there is optimal way to generate this, could you take a look at it ?

Reply:
https://github.com/elastic/elastic-charts/blob/5a9ec7d34a11876766c53c10fea00b4305567189/stories/utils/utils.ts

Reply:
> https://github.com/elastic/elastic-charts/blob/5a9ec7d34a11876766c53c10fea00b4305567189/stories/utils/utils.ts

I am not able to import methods from the utils. That is why I implemented such code by referring to code from elastic/charts [code](https://github.com/elastic/elastic-charts/blob/master/src/chart_types/partition_chart/layout/utils/calcs.ts#L24), and color combinations from [here](https://github.com/elastic/elastic-charts/blob/master/src/mocks/hierarchical/palettes.ts#L22). It would be better if there is anyway I can directly import their utils. Please let me know if you have idea on how to import